package.json

```diff
@@ -34,10 +34,13 @@
   "author": "The Betaflight open source project.",
   "license": "GPL-3.0",
   "dependencies": {
+    "follow-redirects": "^1.4.1",
```

Comment:
These don't seem to be used at runtime. They should be in `devDependencies`.

Reply:
My bad, totally forgot to move them.

Reply:
`npm install --save-dev` can help. :wink: